src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -66,4 +66,9 @@ public CommandResult execute() {
         }
     }
 
+    @Override
+    public boolean isMutating() {
```

Comment:
Remember to update with header comments as the overriden method has a different behavior from the default implementation.

Reply:
noted with thanks.